### PR TITLE
FQDN check should pass/fail after retrying mutliple times to reach the node

### PR DESCRIPTION
**Pull request by ArvinthC3000**
### :nut_and_bolt: Description: What code changed, and why?
As an Automate HA user, when I try to run the verify command with my configuration, the FQDN error is generic. It should be detailed on the error

### :chains: Related Resources
Jira: https://chefio.atlassian.net/browse/CHEF-6048

### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable
Demo Video: https://progresssoftware.sharepoint.com/:v:/s/ChefCoreC/EZdDU_9hw0BEgZEA0p401RgBBCeJCmQRTfDEqMhkzL-fJw?e=qV3RKz

### CODE REVIEW

**Review by jayvikramsharma1 — APPROVED** [review]

File: components/automate-cli/pkg/verifyserver/server/api/v1/fqdn.go [hunk 1]
```diff
@@ -37,6 +37,6 @@ func (h *Handler) CheckFqdn(c *fiber.Ctx) error {
 		return fiber.NewError(http.StatusBadRequest, "node_type should be automate or chef-infra-server, Please provide node_type.")
 	}
 
-	res := h.FqdnService.CheckFqdnReachability(*req, constants.DEFAULT_HTTPS_PORT, time.Minute*1)
+	res := h.FqdnService.CheckFqdnReachability(*req, constants.DEFAULT_HTTPS_PORT, time.Second*30)
```
Comment (Dmaddu) [inline components/automate-cli/pkg/verifyserver/server/api/v1/fqdn.go:40]: Are you sure on this, this impacts the node reachability check

**Review by Dmaddu — CHANGES_REQUESTED** [review]

File: components/automate-cli/pkg/verifyserver/server/api/v1/fqdn.go (hunk 1 above)
Comment (ArvinthC3000) [reply components/automate-cli/pkg/verifyserver/server/api/v1/fqdn.go:40]: The API timeout was set to 35 seconds, and checking for reachablity for 60 seconds, even if the FQDN was reachable after 35 second, the API request might have ended already. So based on that I have reduced it to 30 sec.

File: components/automate-cli/pkg/verifyserver/constants/fqdn.go [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
-	CERTIFICATE_ERROR_MESSAGE      = "FQDN certificate is not valid."
-	CERTIFICATE_RESOLUTION_MESSAGE = "Generate new valid certificates and provide those."
-	IP_TO_HASH_FAIL_MESSAGE        = "Failed to hash the IP."
-	DEFAULT_HTTPS_PORT             = "443"
-	SERVER_IP_HEADER_KEY           = "x-server-ip"
-	CHAN_RESULT_ERROR_MESSAGE      = "error recieved"
-	MIN_NUMBER_OF_CALLS            = 50
+	FQDN_TITLE                           = "FQDN is reachable"
+	FQDN_ERROR_MESSAGE                   = "FQDN is not reachable"
+	INVALID_CERTIFICATE_ERROR            = "FQDN is not reachable"
```
[the remark this answers is not in the document]
Comment (ArvinthC3000) [reply components/automate-cli/pkg/verifyserver/constants/fqdn.go]: Removed the unused one

**Review by Dmaddu — APPROVED** [review]

**Review by Dmaddu — DISMISSED** [review]
pipelines are failing, could you please fix it

**Review by punitmundra — APPROVED** [review]

**Pull request merged**